app/views/get_a_teacher_relocation_payment/claims/application_route.html.erb

```diff
@@ -0,0 +1,68 @@
+<% content_for(
+  :page_title,
+  page_title(
+      t(
+        "get_a_teacher_relocation_payment.forms.application_route.questions.which_route"
+      ),
+      journey: current_journey_routing_name,
+      show_error: @form.errors.any?
+  )
+) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render("shared/error_summary", instance: @form) if @form.errors.any? %>
+
+    <%= form_for @form, url: claim_path(current_journey_routing_name) do |f| %>
```

Comment:
```suggestion
    <%= form_for @form, url: claim_path(current_journey_routing_name), builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
```
The formbuilder has been installed in this app for a long time but never used. Would be good to actually start using it and therefore it should simplify a lot of the view code. I'm working on removing all the custom code we have built up by not using GOVUK formbuilder at the moment

Reply:
👍 yeah makes sense, I'm not planning on merging this pr!